app/views/govuk_component/previous_and_next_navigation.raw.html.erb

```diff
@@ -0,0 +1,18 @@
+<nav class="pagination govuk-previous-and-next-navigation" role="navigation" aria-label="Pagination">
+  <ul class="group">
+    <% if local_assigns.include?(:previous_element) %>
+      <li class="previous-element">
+        <a href="<%= previous_element['href'] %>" rel="previous" title="<%= previous_element['title'] %>">
```

Comment:
What will this do if, say, you pass in the `href` and `label`, but no `title`? I can imagine, for example, a case where the link text is sufficient on its own:

    {"href" => "/kittens/adopting", "label" => "Next: adopting a kitten"}

It looks to me like that would make the link render with a blank `title` attribute: in this case, would a screen reader still read out the link text?